screener/models.py

```diff
@@ -281,6 +282,7 @@ def has_benefit(self, name_abbreviated):
             "wic": self.has_wic,
             "nc_wic": self.has_wic,
             "snap": self.has_snap,
+            "sunbucks": self.has_sunbucks or self.has_tanf or self.has_snap or self.has_medicaid or self.has_medicaid_hi,
```

Comment:
Can you move `self.has_tanf or self.has_snap or self.has_medicaid or self.has_medicaid_hi` to the `SunBucks` calculator. this method it meant to determine if the user already has the benefit. If they have TANF, SNAP, or Medicaid, they are not eligible for the program which is different than the use already having the benefit. An example of the calculator accounting for benefits the user already is eligible for is [EmergencyMedicaid](https://github.com/Gary-Community-Ventures/benefits-api/blob/main/programs/programs/co/medicaid/emergency/calculator.py).

Reply:
> this method it meant to determine if the user already has the benefit

This program (as per its [ticket's](https://github.com/Gary-Community-Ventures/benefits-api/issues/644) requirements) should show up if the user hasn't inputted in step 8 that they are already receiving TANF, SNAP, Medicaid, or Sun bucks itself.

Reply:
The `has_benefit` method is used if the user already has a benefit. If the user has TANF, they do not have Sun Bucks. Instead, call `self.screen.has_benefit('tanf')` in the `SunBucks` calculator.